py/desispec/desi_proc_dashboard.py

```diff
@@ -0,0 +1,383 @@
+import argparse
+import os,glob
+import fitsio
+import astropy.io.fits as pyfits
+from astropy.io import fits
+import subprocess
+import pandas as pd
+import time,datetime
+import numpy as np
+import psycopg2
+import hashlib
+import pdb
+import psutil
+from os import listdir
```

Comment:
Some of these imports aren't used anymore.  Let's remove them to simplify dependencies and load time.  e.g. astropy.io.fits, pandas, psycopg2, hashlib, pdb, subprocess.

Reply:
Done